doc/manual/oqum/help.txt

```diff
@@ -71,7 +71,7 @@ optional arguments:
   --export-outputs CALCULATION_ID TARGET_DIR, --eos CALCULATION_ID TARGET_DIR
                         Export all of the calculation outputs to the specified
                         directory
-  -e, --exports        Comma-separated string specifing the export formats,
+  --exports, -e         Comma-separated string specifing the export formats,
                         in order of priority
-  -l {debug, info, warn, error, critical}, --log-level {debug, info, warn, error, critical}
+  --log-level {debug, info, warn, error, critical}, -l {debug, info, warn, error, critical}
                         Defaults to "info"
```

Comment:
I do not understand the change in the order. Doing `oq engine --help` I get the previous order, first the short option and then long option.